change-notes/1.21/analysis-javascript.md

```diff
@@ -19,5 +19,6 @@
 |--------------------------------|------------------------------|---------------------------------------------------------------------------|
 | Expression has no effect       | Fewer false-positive results | This rule now treats uses of `Object.defineProperty` more conservatively. |
 | Useless assignment to property | Fewer false-positive results | This rule now ignore reads of additional getters. |
+| ZipSlip                        | More results                 | This rule now considers more libraries, including tar as well as zip. |
```

Comment:
The first field should be the `@name` of the query, so "Arbitrary file write during zip extraction ("Zip Slip")" in this case.